xmtp_mls/src/groups/device_sync/backup.rs

```diff
@@ -1,3 +1,4 @@
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
No need for this here. Move this above the `mod backup` line in `device_sync.rs`